nodepool/nl02.sjc1.vexxhost.zuul.ansible.com.yaml

```diff
@@ -186,7 +186,7 @@ providers:
               - net2
               - public
           - name: iosxrv-7.0.2
-            flavor-name: v1-standard-4
+            flavor-name: v2-standard-8
```

Comment:
I am not sure v2 flavors are in this region, you might need to use openstack flavor list to see

Reply:
v1-standard-8

lets use that